package.json

```diff
@@ -6,17 +6,16 @@
   "dependencies": {
     "express": "^4.14.0"
   },
+  "peerDependencies" : {
+    "bs-platform": "file:../bucklescript.git/"
+  },
   "devDependencies": {
-    "bs-platform": "^1.0.1"
+    "bs-platform": "file:../bucklescript.git/"
```

Comment:
After reading https://nodejs.org/en/blog/npm/peer-dependencies/ I think  peer dependencies could be great. In each package there should be only one version of BuckleScript and this peer dependency will enforce a minimal version for the installed BuckleScript. It will not install it though. 

If we put BuckleScript in `dependencies` we could end up with multiple installation of BuckleScript, if we don't put BuckleScript in `peerDependencies` we won't get any warning about version mismatch.